code/modules/mob/living/simple_animal/familiars/familiars.dm

```diff
@@ -104,26 +104,27 @@
 
 /mob/living/simple_animal/familiar/goat
 	name = "goat"
-	desc = "An elder looking goat."
+	desc = "An sprightly looking goat."
```

Comment:
As far as I know, the correct way to do it is 'a sprightly'? An sprightly reads wrong to me anyways.